content/actions/using-containerized-services/about-service-containers.md

```diff
@@ -23,7 +23,7 @@ topics:
 
 Service containers are Docker containers that provide a simple and portable way for you to host services that you might need to test or operate your application in a workflow. For example, your workflow might need to run integration tests that require access to a database and memory cache.
 
-You can configure service containers for each job in a workflow. {% data variables.product.prodname_dotcom %} creates a fresh Docker container for each service configured in the workflow, and destroys the service container when the job completes. Steps in a job can communicate with all service containers that are part of the same job.
+You can configure service containers for each job in a workflow. Service containers are not meant to be used inside composite actions and are not supported to be run inside composite actions. {% data variables.product.prodname_dotcom %} creates a fresh Docker container for each service configured in the workflow, and destroys the service container when the job completes. Steps in a job can communicate with all service containers that are part of the same job.
```

Comment:
Thanks for this update, @nikola-jokic! I'm just going to move the sentence to the end of this paragraph and simplify the wording a little. 

```suggestion
You can configure service containers for each job in a workflow. {% data variables.product.prodname_dotcom %} creates a fresh Docker container for each service configured in the workflow, and destroys the service container when the job completes. Steps in a job can communicate with all service containers that are part of the same job. However, you cannot create and use service containers inside a composite action. 
```